packages/compass-indexes/src/components/search-indexes-table/search-indexes-table.tsx

```diff
@@ -163,7 +168,15 @@ export const SearchIndexesTable: React.FunctionComponent<
   onEditIndex,
   status,
   onDropIndex,
+  onPollIndexes,
 }) => {
+  useEffect(() => {
+    const id = setInterval(onPollIndexes, POLLING_INTERVAL);
```

Comment:
I'm wondering if we shouldn't rather use a setTimeout that we only start once the polling finishes. Just wondering what would happen if search indexes gets slower than 5 seconds yet every 5 seconds we kick off another poll..

Reply:
if it gets slower, then the current one will be still running and the status is `POLLING` and then when fetching we check and skip another fetch